lib/service/awsoidc.go

```diff
@@ -307,7 +310,11 @@ func (updater *AWSOIDCDeployServiceUpdater) updateAWSOIDCDeployService(ctx conte
 		}
 	}()
 
-	updater.Log.DebugContext(ctx, "Updating AWS OIDC Deploy Service", "integration", integration.GetName(), "region", awsRegion)
+	updater.Log.DebugContext(ctx, "Updating AWS OIDC Deploy Service",
+		"integration", integration.GetName(),
+		"region", awsRegion,
+		"version", teleportVersion,
```

Comment:
Can we please be clear about which version this is referring to?
It should be the version we are updating to and not the one we are updating from.

Reply:
changed to "update_version"